### PR TITLE
Issue 1847: Set parent version to test uprade of commons-beanutils

### LINKED ISSUE #1847: Upgrade commons-beanutils from 1.9.3 to 1.9.4
# Task Description

We need to upgrade `commons-beanutils:commons-beanutils:1.9.3` to `1.9.4`, as it fixes security vulnerabilites.

An automated pull request ( strongbox/strongbox-parent#78 )  was raised by `dependabot`.

# Tasks

The following tasks will need to be carried out:
* [ ] Update the version in the [strongbox-parent] project.
* [ ] Apply any required code changes, or dependency excludes.
* [ ] Test and confirm that there are no issues with the core projects ([strongbox], [strongbox-web-integration-tests], etc) after upgrading.
* [ ] Test against the `master` with the steps above.
* [ ] Test against the `issues/1649` with the steps above and create a pull request for for this against the `issues/1649` branch.

# Task Relationships

This task:
* Relates to: strongbox/strongbox-parent#78

# Help

* Check our article on [how to upgrade dependencies and plugins].
* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @carlspring
  * @sbespalov
  * @steve-todorov

[strongbox]: https://github.com/strongbox/strongbox/
[strongbox-parent]: https://github.com/strongbox/strongbox-parent/
[strongbox-web-integration-tests]: https://github.com/strongbox/strongbox-web-integration-tests/
[how to upgrade dependencies and plugins]: https://strongbox.github.io/developer-guide/upgrading-dependencies-and-plugins.html

### PR DESCRIPTION
**Pull request by flaix**
# Pull Request Description

This pull request closes #1847

# Acceptance Test

* [x] Building the code with `mvn clean install -Dintegration.tests` still works.
* [x] Running `mvn spring-boot:run` in the `strongbox-web-core` still starts up the application correctly.
* [x] Building the code and running the `strongbox-distribution` from a `zip` or `tar.gz` still works.
* [x] The tests in the [`strongbox-web-integration-tests`](https://github.com/strongbox/strongbox-web-integration-tests/) still run properly.

# Questions

* Does this pull request break backward compatibility? 
  * [ ] Yes
  * [x] No

* Does this pull request require other pull requests to be merged first? 
  * [ ] Yes, please see #...
  * [x] No

* Does this require an update of the documentation?
  * [ ] Yes, please see strongbox/strongbox-docs#{PR_NUMBER}
  * [x] No

# Code Review And Pre-Merge Checklist

* [x] My code follows the [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html) of this project.
* [x] I have performed a self-review of my own code.
* [ ] I have commented my code in hard-to-understand areas.
* [x] My changes generate no new warnings.
* [ ] I have added tests that prove my fix is effective or that my feature works.
* [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Comment by carlspring** [comment]
I've triggerred our complete build pipeline [here](https://jenkins.carlspring.org/blue/organizations/jenkins/strongbox%2Fbuilds%2Fstrongbox/detail/PR-1951/2/pipeline). Let's see how it goes.

**Comment by flaix** [comment]
I'm sorry, you will probably have to trigger it again. The first commit was missing most of the updated POM files. I have added them in a second commit.

**Comment by steve-todorov** [comment]
@carlspring shouldn't this be targeting strongbox:issues/1649 instead of strongbox:master?

**Comment by flaix** [comment]
No, this is to test the dependency update on the master branch. I used the `issue-1847` branch name for it.

#1953 is to test this update on the `issues/1649` branch. Since this PR is already using the `issue-1847` branch name, I had to use the `issues/1847` name for the new branch off `issues/1647`.

So you can run tests for this, to see if it tests clean for master, and run tests on #1953 to see if it could also be applied to `issues/1649`. It doesn't for me, as I had mentioned in #1847, but that is with or without the dependency update.

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <parent>
         <groupId>org.carlspring.strongbox</groupId>
         <artifactId>strongbox-parent</artifactId>
-        <version>1.0-SNAPSHOT</version>
+        <version>1.0-issue-1847-SNAPSHOT</version>
```
Comment (steve-todorov) [inline pom.xml:11]: Version is wrong - should be the same as what you've set in the `strongbox-parent`. :) 
```suggestion
        <version>1.0-PR-62-issues-1847-SNAPSHOT</version>
```

Comment (flaix) [reply pom.xml:11]: But that *is* the version set in strongbox-parent. This PR is against ˋmasterˋ. #1953 is the PR against ˋissues/1649ˋ

**Comment by flaix** [comment]
This is superseded by PR #1954

**Pull request closed without merging**